src/design-system/stylesheets/main.scss

```diff
@@ -276,6 +276,7 @@ pre {
   overflow: auto;
   border: 1px solid $govuk-border-colour;
   background-color: $app-light-grey;
+  font-size: 16px;
```

Comment:
Could this use `govuk-font()`

Reply:
Since this is a monospaced code example, I don't think we want to use this here.